ansible/examples/builder.yml

```diff
@@ -486,6 +487,43 @@
         update_cache: yes
       when: ansible_os_family == "Suse"
 
+    ## UBUNTU PODMAN TASKS
```

Comment:
are these OK to run on non-ubuntu?  If not should they have a when: ansible_os_family == "Ubuntu" or whatever?

Reply:
They are actually safe to run on RPM-based builders.  I updated the block's comment.